test/start_server_test.js

```diff
@@ -51,4 +52,19 @@ suite('startServer', () => {
       .end(done);
   });
 
+  test('unknown cmd parameter should not throw exception', (done) => {
+	var argv = process.argv;
```

Comment:
this is fine, but if you wanted to pass optional args to `run()` that would be a little cleaner.